src/Web/Slack/Common.hs

```diff
@@ -19,6 +20,10 @@ module Web.Slack.Common
     , mkHistoryReq
     , HistoryRsp(..)
     , Message(..)
+    , Color
+    , unColor
+    , UserId
+    , unUserId
```

Comment:
i didn't want the first syntax with `..` because it exports also the data constructor, and it's great to be able to say that only slack-web can build a `UserId`: if somewhere in your app you have a `UserId` then you know that user is known to slack. It's a 'validated' user id (same for the color: you know it's a validated color, in a format slack considers valid). as for the second syntax `UserId(unUserId)`, it still offers that feature: compared to `..`, it does not export the data constructor. I was just not aware of it :-)

so i've now changed to using the `UserId(unUserId)` syntax, thanks for the tip! (note: in queries, if we want to filter by user, we should accept `Text` not `UserId`, to let the user search for a user id as they like. But when we return from the slack API then we return with the `UserId` type.

Actually you've made me wonder, maybe I should even change to a `!Text` inside the newtype, so it's strict.. But I didn't do it so far.